### PR TITLE
Add footer

### PR DESCRIPTION
**Pull request by volkov97**
Footer implemented and pager padding fixed.

### CODE REVIEW

File: common.blocks/footer/footer.bemtree.js [hunk 1]
```diff
@@ -1,5 +1,32 @@
...
+                text: i18n(block, 'source-code'),
+                icon: 'code'
+            },
+            {
+                url: '//github.com/bem/bem-forum/issues',
+                text: i18n(block, 'error-report'),
+                icon: 'bug'
+            }
+        ].map(link => ({
+            block: 'button',
```
Comment (tadatuta) [inline common.blocks/footer/footer.bemtree.js]: let's try to implement it with `link` block with `icon` inside

File: common.blocks/footer/footer.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
+.footer {
+    &__links {
```
Comment (tadatuta) [inline common.blocks/footer/footer.css]: I think these rules should be applied to footer itself

File: common.blocks/footer/footer.deps.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+({
+    mustDeps: 'i-bem-dom',
```
Comment (tadatuta) [inline common.blocks/footer/footer.deps.js]: what is the reason for `i-bem-dom`?

File: common.blocks/pager/pager.css [hunk 4]
```diff
@@ -0,0 +1,4 @@
+.pager {
```
Comment (tadatuta) [inline common.blocks/pager/pager.css]: let's mix pager with `issues__pager` and move these rules as margins to that element

**Comment by volkov97** [comment]
thanks, all remarks were fixed

**Comment by skad0** [comment]
Please, extract changes for issues pagination in different commit

**Comment by volkov97** [comment]
Done - extracted to separate commit

**Comment by tadatuta** [comment]
🎉

**Pull request merged**